recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "3.2.0rc0" %}
+{% set version = "3.3.0rc0" %}
 
 package:
   name: iris
   version: {{ version }}
 
 source:
   url: https://github.com/SciTools/iris/archive/v{{ version }}.tar.gz
-  sha256: 03589b29d8e20510614ba5d8cf57ce1fca4cf2b128033180beee7e6bf9def202
+  sha256: 9d237efa3d7985f006cbfbfa9b1d329f91a6570e65709ee4e6d580144714092d
```

Comment:
We should be replicating the new standard SciTools practice of sourcing from PyPI. You can replicate the pattern we have in cf-units:

https://github.com/conda-forge/cf_units-feedstock/blob/73b9f1d2b7ef20b79c44a545e99d24a1d70c1018/recipe/meta.yaml#L1-L10

Reply:
I think the `sha256` on PyPI will also be different than the one from GitHub